packages/oscal-react-library/src/components/OSCALControl.tsx

```diff
@@ -43,6 +69,8 @@ function ControlsList(props) {
     fragmentPrefix,
   } = props;
   return (
+    // TODO: I think that in this `control.parts` map is where we need to stuff to make the
+    // last bit of text use the `WithdrawnControlText` component? But I am not 100% sure.
```

Comment:
This also might be wrong? Not entirely sure. `OSCALControlCard` just has the text decoration applied to the whole card which helps (and actually may have implicitly marked children as withdrawn). But in the list view, that's not there to help us (no card) so I *think* this is correct.

I _really_ don't want to just apply this styling to the `Accordion` because that's too focused on the implementation and means we wouldn't be able to display withdrawn controls in other contexts outside of the accordion. So I think the right fix is to make control parts wiser about withdrawn stuff?